cloudinary/cloudinary_wordpress
# Fix/inline transformations

- Fixes detection of inline transformations;
- Fixes duplication of inline transformations;

## QA notes

- In the context of adding an asset to a post, Import an asset from Cloudinary to WP media library with transformations applied, for instance [alternated or combined]:
  - Effect Red 100;
  - Quality Auto Eco;
- Another test is to set the transformations via Media Library;
- Make minor changes and save the post multiple times;
- Either the BE or FE should break, and the data stored in the content of the post shouldn't get weird arguments on those images;
